include/state-observation/tools/rigid-body-kinematics.hpp

```diff
@@ -388,7 +388,7 @@ class Orientation
 
   inline Orientation & setRandom();
 
-  template<typename t>
+  template<typename t = Quaternion>
   inline Orientation & setZeroRotation();
 
   inline Orientation & setZeroRotation();
```

Comment:
This function shouldn't be required since it's calling `setZeroRotation<Quaternion>()`